deployments/arbitrum/usdc/migrations/1679020486_configurate_and_ens.ts

```diff
@@ -0,0 +1,355 @@
+import { BigNumber, Contract, utils } from 'ethers';
+import { expect } from 'chai';
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { calldata, exp, getConfigurationStruct, proposal } from '../../../../src/deploy';
+
+const ENSName = 'compound-community-licenses.eth';
+const ENSResolverAddress = '0x4976fb03C32e5B8cfe2b6cCB31c09Ba78EBaBa41';
+const ENSSubdomainLabel = 'v3-additional-grants';
+const ENSSubdomain = `${ENSSubdomainLabel}.${ENSName}`;
+const ENSTextRecordKey = 'v3-official-markets';
+
+// https://github.com/OffchainLabs/arbitrum/blob/master/packages/arb-bridge-eth/contracts/libraries/AddressAliasHelper.sol
+function applyL1ToL2Alias(l1Address: string) {
```

Comment:
I think we should move all these helpers into a separate util file to be shared with future migrations.

Reply:
moved into `scenario/utils/arbitrumUtils.ts`